### DISCUSSION TITLE
Make favoriting maps more accessible in UI

### DISCUSSION THREAD

**Discussion by yamplum** [ppy/osu#21928] [Ideas] [answered]
It would be nice to have a more easily accessible "Add beatmap to favorites" button, in particular right on the song select screen, as well as on the results screen. My highly technical concept mockup: 
![image](https://user-images.githubusercontent.com/118788705/209965110-02a4def0-951d-44fb-b784-dd77503b9651.png)

I've started favoriting more maps recently and having to (put my pen down and pick up the mouse ->) right click -> details -> favorite is a bit of friction that adds up over time. I'm also often unsure if I've already favorited the map or not at a glance.

Could potentially encourage more people to engage with maps as a community? The numbers seem somewhat skewed right now. 
![image](https://user-images.githubusercontent.com/118788705/209965901-c3858969-1c31-422f-b26b-56ee9f7fd1d0.png)

**Comment by peppy** [comment, marked as answer]
Very much agree this should be on song select at very least. We can revisit after the redesign is implemented. @arflyte highlight in case you're watching.

**Comment by arflyte** [comment]
Funny, I thought I already had this in the design lol. Will add it.
